.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -86,6 +86,14 @@ parameters:
   displayName: Build 1804 FIPS Gen2 GPU+containerd
   type: boolean
   default: true
+- name: build2004fipscontainerd
+  displayName: Build 2004 FIPS containerd
+  type: boolean
+  default: true
```

Comment:
let's use false here before we finalize everything and want to include this vhd build in weekly aks vhd build.